modules/tinymce/tools/docs/tinymce.EditorManager.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the base class for the link:../tinymce.root/[`tinymce`] global, a factory for link:../tinymce.editor/[`tinymce.Editor`] instances.
+ *
+ * The global link:../tinymce.root/[`tinymce`] is the only one expected to be created.
```

Comment:
```suggestion
 * The global link:../tinymce.root/[`tinymce`] is the only EditorManager instance expected to be created.
```
This clears it up I think, as `one` I thought was referring to editor instances and what caused part of my previous confusion.

Reply:
@brian-forte might also have some other suggestions? We probably should have included him as a reviewer on this PR anyways given it's for docs.